### ISSUE TITLE
Need updated client_api mustache file to handle Python 2 long integers

### ISSUE THREAD

**Issue by tenortim** [Isilon/isilon_sdk#32]
The version of the swagger Python mustache we're using is rather old and has a number of issues. Specifically in this case, it fails to handle Python 2 longs, so any API calls with larger integers e.g. setting quotas fail thus:

```
Traceback (most recent call last):
  File "provtest.py", line 143, in provision_storage
    quota = task.create_quota()
  File "C:\Users\timw\test\provtest\modules\provisioning_task.py", line 382, in create_quota
    self.quota_id = self.api_client.create_quota(self.share_path, advisory=self.size)
  File "C:\Users\timw\test\provtest\modules\isi_api_client_8_0.py", line 59, in create_quota
    api_response = api_instance.create_quota_quota(quota_quota)
  File "C:\Users\timw\test\test_env\lib\site-packages\isi_sdk_8_0\apis\quota_api.py", line 127, in create_quota_quota
    callback=params.get('callback'))
  File "C:\Users\timw\test\test_env\lib\site-packages\isi_sdk_8_0\api_client.py", line 345, in call_api
    response_type, auth_settings, callback)
  File "C:\Users\timw\test\test_env\lib\site-packages\isi_sdk_8_0\api_client.py", line 157, in __call_api
    body = self.sanitize_for_serialization(body)
  File "C:\Users\timw\test\test_env\lib\site-packages\isi_sdk_8_0\api_client.py", line 242, in sanitize_for_serialization
    for key, val in iteritems(obj_dict)}
  File "C:\Users\timw\test\test_env\lib\site-packages\isi_sdk_8_0\api_client.py", line 242, in <dictcomp>
    for key, val in iteritems(obj_dict)}
  File "C:\Users\timw\test\test_env\lib\site-packages\isi_sdk_8_0\api_client.py", line 242, in sanitize_for_serialization
    for key, val in iteritems(obj_dict)}
  File "C:\Users\timw\test\test_env\lib\site-packages\isi_sdk_8_0\api_client.py", line 242, in <dictcomp>
    for key, val in iteritems(obj_dict)}
  File "C:\Users\timw\test\test_env\lib\site-packages\isi_sdk_8_0\api_client.py", line 238, in sanitize_for_serialization
    for attr, _ in iteritems(obj.swagger_types)
AttributeError: 'long' object has no attribute 'swagger_types'
```

This was fixed in upstream by the changes in commit https://github.com/swagger-api/swagger-codegen/commit/89befeb45b86905937e91a20880b1c36537db26e#diff-027e164ab608834aec24a0ee3c3fc9c5

**Comment by daltschu22** [comment]
Yes, this is affecting us aswell.

Thanks!

**Comment by cbrainerd** [comment]
Fixed in by #33 in [0.1.10](https://github.com/Isilon/isilon_sdk/releases/tag/0.1.10)
